extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/PrivilegesBuilder.scala

```diff
@@ -385,8 +414,18 @@ object PrivilegesBuilder {
         inputObjs += databasePrivileges(quote(database))
 
       case "DescribeFunctionCommand" =>
-        val func = getPlanField[FunctionIdentifier]("functionName")
-        inputObjs += functionPrivileges(func.database.orNull, func.funcName)
+        val (db: Option[String], funName: String) =
```

Comment:
shall we directly get a `FunctionIdentifier`  here and generalize it to AuthUtils like `getFunctionIdentifier` if possible?

Reply:
Getting the FunctionIdentifier via the `functionName` field might not be that common, so maybe we can generalize it as a method the next time we need to get the FunctionIdentifier?

Reply:
Yes, alright.